setup.py

```diff
@@ -53,6 +53,13 @@ def build_extensions(self):
                 cgal_dir = DIR / zip_obj.namelist()[0]
                 zip_obj.extractall(DIR)
 
+            # Patch for FastJet core version 3.4.0
+            # To be removed when it is applied upstream
+            subprocess.run(
+                ["sed", "-i", "-E", "-f", DIR / "patch.sed", "pyinterface/fastjet.i"],
```

Comment:
I was between sed and (a literal) patch. We shouldn't have any issues with swig since I am deliberately applying this to the instructions file `fastjet.i` instead of the swig-generated `_swig.py`. Now that I look back at this I agree that an actual patch will be a cleaner solution. Will make it now.

Reply:
changed to `patch`